model/k8s/metrics.yaml

```diff
@@ -102,6 +102,34 @@ groups:
       The actual accuracy would depend on the instrumentation and operating system.
     instrument: gauge
     unit: "s"
+  - id: metric.k8s.node.allocatable.cpu
```

Comment:
Please define the `entity_association` for these metrics. It should be `k8s.node`.

See for example https://github.com/open-telemetry/semantic-conventions/blob/main/model/k8s/metrics.yaml#L507-L508

Reply:
should be added